robolectric-resources/src/main/java/org/robolectric/RuntimeEnvironment.java

```diff
@@ -136,27 +136,27 @@ public static void setMasterScheduler(Scheduler masterScheduler) {
     RuntimeEnvironment.masterScheduler = masterScheduler;
   }
 
-  public static void setSystemResourceLoader(ResourceLoader systemResourceLoader) {
-    RuntimeEnvironment.systemResourceLoader = systemResourceLoader;
+  public static void setSystemResourceProvider(ResourceProvider systemResourceProvider) {
```

Comment:
I think these API changes are of fairly limited impact, and unavoidable in any case, so LGTM. Let's mention it in the release notes. Nit: "compile time" is two words while "runtime" is one, so internal cap if ya want? ;-)